extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -414,114 +409,112 @@ private function getBadgeMsgKeyPart($badgeTypeId) {
 			return 'achievements-badge-desc-edit-plus-category';
 		else
-			return 'achievements-badge-desc-'.$this->getBadgeMsgKeyPart($badgeTypeId);
+			return 'achievements-badge-desc-' . $this->getBadgeMsgKeyPart( $badgeTypeId );
 	}
 
-	public function getBadgePersonalDescKey($badgeTypeId) {
-		$badgeType = $this->getBadgeType($badgeTypeId);
+	public function getBadgePersonalDescKey( $badgeTypeId ) {
+		$badgeType = $this->getBadgeType( $badgeTypeId );
 		$msgKeyPart = null;
 
-		if($badgeType == BADGE_TYPE_INTRACKEDITPLUSCATEGORY)
+		if( $badgeType == BADGE_TYPE_INTRACKEDITPLUSCATEGORY )
 			$msgKeyPart = 'edit-plus-category';
-		elseif($badgeType == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM)
-			return $this->getBadgeDescKey($badgeTypeId);
+		elseif( $badgeType == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM )
+			return $this->getBadgeDescKey( $badgeTypeId );
 		else
-			$msgKeyPart = $this->getBadgeMsgKeyPart($badgeTypeId);
+			$msgKeyPart = $this->getBadgeMsgKeyPart( $badgeTypeId );
 
 		return "achievements-badge-your-desc-{$msgKeyPart}";
 	}
 
-	public function getBadgeDescHoverKey($badgeTypeId) {
-		$badgeType = $this->getBadgeType($badgeTypeId);
+	public function getBadgeDescHoverKey( $badgeTypeId ) {
+		$badgeType = $this->getBadgeType( $badgeTypeId );
 		$msgKeyPart = null;
 
-		if($badgeType == BADGE_TYPE_INTRACKEDITPLUSCATEGORY)
-			$msgKeyPart =  'edit-plus-category';
+		if( $badgeType == BADGE_TYPE_INTRACKEDITPLUSCATEGORY )
+			$msgKeyPart = 'edit-plus-category';
 		else
-			$msgKeyPart = $this->getBadgeMsgKeyPart($badgeTypeId);
+			$msgKeyPart = $this->getBadgeMsgKeyPart( $badgeTypeId );
 
-		if($badgeType == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM)
+		if( $badgeType == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM )
 			return "achievements-badge-desc-{$msgKeyPart}";
 		else
 			return "achievements-badge-hover-desc-{$msgKeyPart}";
 	}
 
-	public function getBadgeToGetKey($badgeTypeId) {
-		if($this->getBadgeType($badgeTypeId) == BADGE_TYPE_INTRACKEDITPLUSCATEGORY)
+	public function getBadgeToGetKey( $badgeTypeId ) {
+		if( $this->getBadgeType( $badgeTypeId ) == BADGE_TYPE_INTRACKEDITPLUSCATEGORY )
 			return 'achievements-badge-to-get-edit-plus-category';
 		else
-			return 'achievements-badge-to-get-'.$this->getBadgeMsgKeyPart($badgeTypeId);
+			return 'achievements-badge-to-get-' . $this->getBadgeMsgKeyPart( $badgeTypeId );
 	}
 
-	public function getBadgeToGetDetailsKey($badgeTypeId) {
-		$badgeType = $this->getBadgeType($badgeTypeId);
+	public function getBadgeToGetDetailsKey( $badgeTypeId ) {
+		$badgeType = $this->getBadgeType( $badgeTypeId );
 		$msgKeyPart = null;
 
-		if($badgeType == BADGE_TYPE_INTRACKEDITPLUSCATEGORY)
-			$msgKeyPart =  'edit-plus-category';
-		elseif($badgeType == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM)
-			$msgKeyPart =  'community-platinum';
+		if( $badgeType == BADGE_TYPE_INTRACKEDITPLUSCATEGORY )
+			$msgKeyPart = 'edit-plus-category';
+		elseif( $badgeType == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)